eng/wpf-debug.targets

```diff
@@ -13,7 +13,10 @@
   <PropertyGroup>
     <WpfConfig Condition="'$(WpfConfig)'==''">Debug</WpfConfig>
     <RuntimeIdentifier>win-$(PlatformTarget)</RuntimeIdentifier>
-    <WPFArtifactsPath>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\'))\artifacts\packaging\$(WpfConfig)\Microsoft.DotNet.Wpf.GitHub.$(WpfConfig)</WPFArtifactsPath>
+    
+    <WpfPackagesCommonPath>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\'))\artifacts\packaging</WpfPackagesCommonPath>
+    <WPFArtifactsPath Condition="'$(PlatformTarget)'=='x86'">$(WpfPackagesCommonPath)\$(WpfConfig)\Microsoft.DotNet.Wpf.GitHub.$(WpfConfig)</WPFArtifactsPath>
+    <WPFArtifactsPath Condition="'$(PlatformTarget)'=='x64'">$(WpfPackagesCommonPath)\$(WpfConfig)\x64\Microsoft.DotNet.Wpf.GitHub.$(WpfConfig)</WPFArtifactsPath>
```

Comment:
How about the `Arm` ?

How about the code below?

```xml
<WPFPlatformTargetArtifacts Condition="'$(PlatformTarget)'=='x64'">x64\</WPFPlatformTargetArtifacts>

<WPFArtifactsPath>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\'))\artifacts\packaging\$(WpfConfig)\$(WPFPlatformTargetArtifacts)Microsoft.DotNet.Wpf.GitHub.$(WpfConfig)</WPFArtifactsPath>
```

Reply:
Right, I missed about ARM64. Thanks @lindexi for pointing that out. I will update the PR.

Reply:
@lindexi, regarding ARM64, it still uses the same directory as AnyCPU, hence we don't need to add a separate clause for that.